client/src/store/comments/reducers.ts

```diff
@@ -109,65 +128,44 @@ export const commentsReducer = (state: IComments = defaultState, action: { type:
                     return info;
                 })
                 :
-                [...state.onChangeComments, { postId: action.payload.postId, comment: action.payload.comment }];
+                [...state.onChangeComments, {postId: action.payload.postId, comment: action.payload.comment}];
             return {
                 ...state,
                 onChangeComments: changedComments,
             };
         case ADD_COMMENT:
-            return {
-                ...state,
-                comments: [{
-                    _id: action.payload._id,
-                    postId: action.payload.postId,
-                    authorId: {
-                        _id: action.payload.authorId._id,
-                        username: action.payload.authorId.username,
-                        photoPath: action.payload.authorId.photoPath,
-                    },
-                    comment: action.payload.comment,
-                    createdAt: action.payload.createdAt,
-                }, ...state.comments],
+            const {
+                authorId: {_id: authorId, username, photoPath}, comment, postId: newPostId,
+                _id: commentId, createdAt,
+            }:
+                {
+                    authorId: { _id: string, username: string, photoPath: string }, comment: string,
+                    postId: string, _id: string, createdAt: string,
```

Comment:
why did u rewrite it into one line? hard to read

Reply:
fixed